src/Components/Task.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, useState } from 'react'
+import {
+    Box,
+    Checkbox,
+    ListItem,
+    Typography
+} from '@material-ui/core'
+
+import DeleteIcon from '@material-ui/icons/Delete';
+
+interface Props {
+    task: {
+        title: string,
+        taskId: number
+    },
+    onRemove: Function
+}
+
+const Task: FC<Props> = ({ task, onRemove }: Props) => {
+
+    const [taskChecked, setTaskChecked] = useState(false)
+
+    function toggleTaskChecked() {
```

Comment:
Use arrow function instead, btw, what is `toggleTaskChecked`?

Reply:
Resolved on 4650a07c585dd6f2c78bbe6baa919bcb496ff3b4